common/constants.js

```diff
@@ -22,23 +22,19 @@ module.exports = Object.freeze({
     MAX_QUEUE_SIZE: 100,
     MAX_SEND_RETRIES: 2
   },
-  ETCD: {
```

Comment:
These must not be removed-- these are used by the rate limiting feature!!

Reply:
They are below : 
https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/304/files/cd4fd9befa3b99fe1d02e53f856f18eaaec17db5#diff-97bb265f5747f3062215bfee484e6d48R326